docs/_docs/snapshots/snapshots.adoc

```diff
@@ -178,9 +172,56 @@ the topology and wait while the data gets rebalanced and indexes are rebuilt.
 |N > M | Unsupported.
 |===
 
+=== Automatic snapshot restore procedure
+
+The automatic restore procedure allows the user to restore cache group(s) from a snapshot on an active cluster using the Java API or Command Language Interface (CLI).
```

Comment:
I think we can directly mention here the `control script` web page which is used for the restore procedure.